erts/emulator/test/port_SUITE_data/file_while_alive.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * %CopyrightBegin%
+ *
+ * SPDX-FileCopyrightText: 2025 Ericsson and contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * %CopyrightEnd%
+ */
+
+/*
+ * Test utility which waits for SIGTERM and responds by deleting a temp file
```

Comment:
The file path is used because it's easy to verify externally, even when pipes to pass messages out of the process may have been rudely crashed.